x-pack/plugins/global_search_bar/public/components/search_bar.tsx

```diff
@@ -0,0 +1,205 @@
+            const titleB = b.title.toUpperCase(); // ignore upper and lowercase
+            if (titleA < titleB) return -1;
+            if (titleA > titleB) return 1;
+            return 0;
+          });
+
+          setOptions(arr);
+        },
+        error: () => {
+          // TODO #74430 - add telemetry to see if errors are happening
+          // Not doing anything on error right now because it'll either just show the previous
+          // results or empty results which is basically what we want anyways
+        },
+        complete: () => {},
+      });
+    },
+    250,
+    [searchValue]
+  );
+
+  const onKeyDown = (event: KeyboardEvent) => {
+    if (event.key === '/' && (isWindows ? event.ctrlKey : event.metaKey)) {
+      if (searchRef) {
+        event.preventDefault();
+        searchRef.focus();
+      }
+    }
+  };
+
+  const onChange = (selected: EuiSelectableTemplateSitewideOption[]) => {
+    // @ts-ignore - ts error is "union type is too complex to express"
+    const { url } = selected.find(({ checked }) => checked === 'on');
+
+    if (/^https?:\/\//.test(url)) window.location.assign(url);
+    else {
+      navigateToUrl(url);
+      (document.activeElement as HTMLElement).blur();
+      if (searchRef) {
+        clearField(searchRef);
+        searchRef.dispatchEvent(blurEvent);
+      }
+    }
+  };
+
+  useEvent('keydown', onKeyDown);
+
+  return (
+    <EuiSelectableTemplateSitewide
+      onChange={onChange}
+      options={options}
+      searchProps={{
+        onSearch: setSearchValue,
+        'data-test-subj': 'header-search',
+        inputRef: setSearchRef,
+        compressed: true,
+        placeholder: i18n.translate('xpack.globalSearchBar.searchBar.placeholder', {
+          defaultMessage: 'Search Elastic',
+        }),
+      }}
+      emptyMessage={
+        <EuiSelectableMessage style={{ minHeight: 300 }}>
+          <p>
+            <FormattedMessage
+              id="xpack.globalSearchBar.searchBar.noResultsHeading"
+              defaultMessage="No results found"
+            />
+          </p>
+          <p>
+            <FormattedMessage
+              id="xpack.globalSearchBar.searchBar.noResults"
+              defaultMessage="Try searching for applications and saved objects by name."
+            />
+          </p>
+        </EuiSelectableMessage>
+      }
+      popoverFooter={
+        <EuiText color="subdued" size="xs">
+          <EuiFlexGroup
+            alignItems="center"
+            justifyContent="flexEnd"
+            gutterSize="s"
+            responsive={false}
+            wrap
+          >
+            <FormattedMessage
+              id="xpack.globalSearchBar.searchBar.shortcut"
+              defaultMessage="{what}{how}"
+              values={{
+                what: <EuiFlexItem grow={false}>Shortcut</EuiFlexItem>,
+                how: (
+                  <EuiFlexItem grow={false}>
+                    <EuiBadge>{isWindows ? 'Ctrl + /' : 'Command + /'}</EuiBadge>
```

Comment:
maybe?
```suggestion
                    <EuiBadge>{isWindows ? 'Ctrl + /' : '⌘ + /'}</EuiBadge>
```

Reply:
Thanks for bringing this up!

This was kind of just done without too much thought but when you brought it up @ryankeairns and I realized spelling out "Command" has much better screen reader support than getting a special character read correctly. So, in that vein, I'm actually also going to be spelling out "Control" here now because "Ctrl" is hard to read without vowels and isn't doing anyone any favors by being a few characters shorter.